.adr/001-dashboard-v1-architecture.md

```diff
@@ -0,0 +1,96 @@
+# Dashboard V1 architecture
```

Comment:
nitpick: the title gives the impression that this doc is about the dashboard architecture in total. Can we narrow down the scope of the title to something like "Dashboard search functionality architecture"?

Reply:
Yeah, the decision's impact will be all over the dashboard, but the decision was about how to implement the search functionality. I will suggest 'Dashboard Catalog' or 'Dashboard Search'

Reply:
yup sounds good 👍